Makefile

```diff
@@ -15,6 +15,10 @@ default:
 	go get -v -d ./...
 	go install -v ./...
 
+client:
```

Comment:
You might want to make this .PHONY; I don't think it matters in this case, but I don't really member all of make's rules...